tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -70,30 +70,36 @@ def convert_value(value, type_):
         #attempt numeric conversion
         try:
             if '.' in value:
-                return float(value)
+                return float(value.strip())
```

Comment:
This is not necessary. `float()` already works even if the passed argument has leading / trailing spaces.

Same below.